File: Examples/CSharp/AsposePdfFacades/Annotations/DeleteAllAnnotations.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace CSharp.AsposePDFFacades.Annotations
{
    public class DeleteAllAnnotations
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Annotations();
            //open document
            PdfAnnotationEditor annotationEditor = new PdfAnnotationEditor();
            annotationEditor.BindPdf(dataDir+ "DeleteAllAnnotations.pdf");
            //delete all annoations
            annotationEditor.DeleteAnnotations("Text");
            //save updated PDF
            annotationEditor.Save(dataDir+ "DeleteAllAnnotations_out.pdf");
            
        }
    }
}

File: Examples/VisualBasic/AsposePdfFacades/Annotations/DeleteAllAnnotations.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades

Namespace AsposePDFFacades.Annotations
    Public Class DeleteAllAnnotations
        Public Shared Sub Run()
            ' ExStart:DeleteAllAnnotations
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Annotations()
            ' Open document
            Dim annotationEditor As New PdfAnnotationEditor()
            annotationEditor.BindPdf(dataDir & Convert.ToString("DeleteAllAnnotations.pdf"))
            ' Delete all annoations
            annotationEditor.DeleteAnnotations()
            ' Save updated PDF
            annotationEditor.Save(dataDir & Convert.ToString("DeleteAllAnnotations_out_.pdf"))
            ' ExEnd:DeleteAllAnnotations
        End Sub
    End Class
End Namespace
